packages/telemetry/src/browser.ts

```diff
@@ -0,0 +1,81 @@
+import type { Span } from '@opentelemetry/api'
+import type { startSpanType, findActiveSpan } from './index'
+import { Telemetry as TelemetryClass, TelemetryNoop } from './index'
+import { WebTracerProvider } from '@opentelemetry/sdk-trace-web'
+import { browserDetectorSync } from '@opentelemetry/resources'
+import { SimpleSpanProcessor } from '@opentelemetry/sdk-trace-base'
+import { OTLPTraceExporter } from './span-exporters/websocket-span-exporter'
+
+declare global {
+  interface Window {
+    __CYPRESS_TELEMETRY__?: {context: {traceparent: string}}
+    cypressTelemetrySingleton?: TelemetryClass | TelemetryNoop
+  }
+}
+
+let telemetryInstance: TelemetryNoop | TelemetryClass = new TelemetryNoop
+
+/**
+ * Initialize the telemetry singleton
+ * @param namespace - namespace to apply to the singleton
+ * @param config - object containing the version
+ * @returns void
+ */
+const init = ({ namespace, config }: { namespace: string, config: {version: string}}): void => {
```

Comment:
Should we throw an error here if `init` has already been called to let the develop know they should be calling `attach` instead?

Reply:
this is done